pkg/sdk/logging/model/output/elasticsearch.go

```diff
@@ -259,6 +259,9 @@ type ElasticsearchOutput struct {
 	DataStreamIlmPolicy string `json:"data_stream_ilm_policy,omitempty"`
 	// Specify whether overwriting data stream ilm policy or not.
 	DataStreamIlmPolicyOverwrite bool `json:"data_stream_ilm_policy_overwrite,omitempty"`
+	// Specify wether to use legacy template or not. (default: true)
+	// +kubebuilder:validation:Optional
+	UseLegacyTemplate *bool `json:"use_legacy_template,omitempty" plugin:"default:true"`
```

Comment:
I beleive we don't have to add ```plugin:"default:true"``` if this is the default in the output anyways. It will make an unnecessary configcheck and reload for existing deployments.

Reply:
Checked once again, this seems to be the default in the fluentd plugin, so we should omit the plugin tag on our side to avoid unecessary config changes in existing deployments: https://github.com/uken/fluent-plugin-elasticsearch/blame/28ade35649cff7a2dd49dd550ac6576cffde8af0/lib/fluent/plugin/out_elasticsearch.rb#L176

Reply:
@aslafy-z in case you missed this one

Reply:
Thank you, I missed it. Applied!